src/unit/engine/UberFilesizeTestEngine.php

```diff
@@ -69,6 +75,20 @@ private function checkFilesizes() {
         }
 
         if (count($large_files) > 0) {
+            if ($skip_lfs_files) {
+                // execute "git lfs" only if there are files over the limit,
+                // because lfs file list might be very big.
+                $lfs_files = $this->getLFSFiles();
+                $non_lfs_large_files = [];
```

Comment:
You can rewrite this with [array_diff](http://php.net/manual/en/function.array-diff.php) into a simple set difference operation.